src/main/python/passage_retrieval/segment.py

```diff
@@ -23,16 +21,22 @@
                     help='number of threads', required=False)
     parser.add_argument("--tokenize", '-t', type=str, default=None,
                     help='defaults to full documents if tokenizer not provided.', required=False)
+    parser.add_argument("--min", '-m', type=int, default=0,
+                    help='minimum size value for tokenizer, defaults to zero', required=False)
     parser.add_argument("--raw", '-r', action='store_true',
                     help='defaults to false for transformed text', required=False)
 
     args = parser.parse_args()
     
+    logfile = os.path.join(os.path.dirname(os.path.abspath(args.output)), 
+                           os.path.basename(os.path.abspath(args.output)) + '.log')
+    
+    logging.basicConfig(level=logging.DEBUG,
```

Comment:
If you want to reuse the logger, you could create a `Logger` class to set these attributes in one place (like `log4j.properties` in Java)?